portfolio/src/main/webapp/index.html

```diff
@@ -30,30 +30,28 @@
     <script src="task.js"></script>
     <script src="calendar.js"></script>
     <script src="request.js"></script>
+    <script src="apiLibrary.js"></script>
     <script src="calendarApi.js"></script>
     <script src="sheetsApi.js"></script>
     <script src="tasksApi.js"></script>
     <script
       async
       defer
       src="https://apis.google.com/js/api.js"
-      onload="handleClientLoadTasks()"
+      onload="handleClientLoad()"
```

Comment:
Which handleClientLoad function is this?

Reply:
```HandleClientLoad()``` is the function that's run when the HTML file loads that start authentication and determines the previous login status.